src/Spinner.component.tsx

```diff
@@ -51,15 +51,18 @@ export class SpinnerComponent extends React.Component<ISpinnerProps, ISpinnerSta
   }
 
   render() {
+    const divStyle = this.props.style ?? { display: 'inline-block' };
```

Comment:
I guess `??` operator was introduced in later typescipr version then used here (2.6.2).

You could use `||` operator, or update typescript dependency.

```suggestion
    const divStyle = this.props.style || { display: 'inline-block' };
```

Reply:
Hi, @Simply007 thank you for interesting in my PR. My local codes fine with '??', I see your code source not compatible with this version after send to PR. After that, I closed this PR and create a new PR with a simple code block. Please check this link > https://github.com/Simply007/react-spinners/pull/2